_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```

Comment:
What's this for?

Reply:
I reverted this part in b6de7558e5e6bfc915ec7deae88e723dea2c8727